js-webapp-async/README.md

```diff
@@ -1,22 +1,26 @@
 # Introduction
-In a [previous blogpost](../js-webapp/README.md) we discussed how to run c++ code on the web using Javascript.
+In an [earlier blogpost](../js-webapp/README.md) we discussed how to run c++ code on the web using Javascript. We created a webapp that executed some C++ code and then showed the result. While the page was running the C++ code, the page was blocked and unresponsive. That was not a problem then, because the computation done in the code was very quick. Blocked UI becomes a problem when we are performing tasks that take a bit longer to run. 
 
-We created a webapp that executed some C++ code and showed the result. While the page was running the C++ code, the page was blocked and unresponsive. That was not a problem then, because the computation done in the code was tiny. This becomes a problem when we are performing long running tasks instead. How to prevent blocking when running long running tasks in c++?
+How to prevent blocking when running long running tasks in c++?
 
-In this blogpost, we will use web workers to solve this problem by running the code asynchronously.
+In this blogpost, we will use web workers to solve this problem by running the code in another thread.
 
 ### Long-running tasks with web worker
 
-Let's have a look at the code we ended up with in last blog. When loading the page, the webassembly is excecuted, after which the page can finish rendering. Because the webassembly code was very quick, this was fine. For this blog we assume we have a more long running task. We create such task artificially by adding a few seconds of sleep in the c++ code (TODO show snippet). Like in the previous post, we compile the c++ code to create webassembly. The resulting page with our slow task is now here (TODO link to sync demo).
+Let's have a look at the code we ended up with in last blog. When loading the page, the webassembly is excecuted, after which the page can finish rendering. Because the webassembly code was very quick, this was fine. For this blog we assume we have a longer running task. We create such task artificially by adding a few seconds of sleep in the [c++ code](newtonraphson.cpp). Like in the previous post, we compile the c++ code to create webassembly. The resulting page with our slow task is now [here](https://nlesc-jcer.github.io/run-cpp-on-web/js-webapp-async/example-blocking.html).

```

Comment:
```suggestion
Let's have a look at the code we ended up with in the last blog. When loading the page, the WebAssembly code is executed, after which the page can finish rendering. Because the WebAssembly code was very quick, this was fine. For this blog, we assume we have a longer running task. We create such task artificially by adding a few seconds of sleep in the [c++ code](newtonraphson.cpp). Like in the previous post, we compile the C++ code to create WebAssembly code. The example page with our slow task is can be found [here](https://nlesc-jcer.github.io/run-cpp-on-web/js-webapp-async/example-blocking.html).
```